packages/widgets/src/lib/components/InferenceWidget/shared/WidgetOutputConvo/WidgetOutputConvo.svelte

```diff
@@ -1,14 +1,18 @@
+<script context="module" lang="ts">
```

Comment:
"<script context="module" lang="ts">" is a nice way to do it. However, I think in this codebase, we would create a separate file:
`..widgets/src/lib/components/InferenceWidget/shared/WidgetOutputConvo/types.ts` (don't remember if it is Types.ts or types.ts)

Reply:
Even better! Thanks for the tips :) Updated in https://github.com/huggingface/huggingface.js/pull/493/commits/5bd519299ce3bc736f30facce4fca33b57b6fb2e (as a matter of fact we already have a `types.js` module in the shared folder)